src/Relay.csproj

```diff
@@ -93,16 +93,9 @@
 
     <ItemGroup>
       <PackageReference Include="Humanizer" Version="2.14.1" />
+        <PackageReference Include="DynamoVisualProgramming.Revit" Version="3.*" />
```

Comment:
I don't really want to reference a Revit - specific package as I want RElay to work in civil 3d one day. But we can do this for now.